package.json

```diff
@@ -60,6 +62,7 @@
     "jsonwebtoken": "^9.0.0",
     "jwks-rsa": "^3.0.1",
     "msal": "^1.4.18",
+    "primeicons": "^6.0.1",
```

Comment:
where is this package being used ?
If it is not used, don't add it in package.json

Reply:
This is needed for prime react icons

Reply:
Which prime react icon are u using and why ?
Ideally all the icons should come from Arun.

Reply:
ok then we can remove